engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/meta/GetSourceLocationNode.java

```diff
@@ -0,0 +1,51 @@
+package org.enso.interpreter.node.expression.builtin.meta;
+
+import com.oracle.truffle.api.CompilerDirectives.TruffleBoundary;
+import com.oracle.truffle.api.TruffleException;
+import com.oracle.truffle.api.TruffleStackTrace;
+import com.oracle.truffle.api.nodes.Node;
+import org.enso.interpreter.dsl.BuiltinMethod;
+import org.enso.interpreter.runtime.data.text.Text;
+import org.enso.interpreter.runtime.error.PanicException;
+
+@BuiltinMethod(
+    type = "Meta",
+    name = "get_source_location",
+    description = "Returns a textual representation of the location of the callsite.")
+public class GetSourceLocationNode extends Node {
+
+  Text execute(Object _this, long framesToSkip) {
```

Comment:
the `framesToSkip` argument is going to be enso-facing (i.e. its name becomes enso-side arg name) so please make it snake_case.